### DISCUSSION TITLE
Bad attack config. Don’t expect your bot to attack.

### DISCUSSION THREAD

**Discussion by Wartortle37** [isid0re/SoloLeveling#65] [Q&A] [answered]
Need help. When I try to start bot it logs in but says Bad attack config. Dont expect your bot to attack. It just stay where it spawns.

**Comment by isid0re** [comment, marked as answer]
If it says bad attack config. There are 2 possible causes that I am aware of.

1. You are missing all or some of the autobuild template files that need to be installed into /libs/config/builds
2. You are using a customized entry/loader script that is ignoring your sololeveling config file.

**Reply by Wartortle37** [reply]
I can’t seem to get it to work. Could you please explain it to me like I’m 5 and tell me where each file for the Sololeveling-master folder should go into. I just downloaded kolbot so there should be no customized entry/loader script.

**Reply by isid0re** [reply]
Sure. I can try. The easiest way I can think of explaining it is highlight the 3 folders from sololeveling-master (folders are bots , config, and sololeveling) and drag them into the kolbot libs folder.

There will already be a config and bots folder there but that's ok. All we are doing is copying into those existing folders.

After you are done copying the libs folder will show 1 new folder called sololeveling with 3 folders inside.

Inside the bots folder you will have a new file called sololeveling.js

Inside the config file click the builds folder and you will see a character build files for the currently supported classes (paladin, sorceress, necromancer, and druid). The filenames will be like paladin.start or sorceress.blizzard etc.

And finally you will have 4 config files added to the config folder. One for each supported class. Paladin.sololeveling.js sorceress.sololeveling.js etc...

After you have verified the installed files are all there. You will need to log into your account and make your character. After making the character you will need to rename the class specific config file to that characters name. For example, 
If your character is a paladin called mchammerz you would rename the paladin.sololeveling.js file to paladin.mchammerz.js 

Finally you would start your profile in kolbot and let it run. 

Hope that helps.

**Reply by Wartortle37** [reply]
Thank you so much. It works now
